package.json

```diff
@@ -15,7 +15,7 @@
     "@babel/runtime": "^7.7.7",
     "@react-native-community/cli": "^3.0.4",
     "@wordpress/babel-preset-default": "file:./gutenberg/packages/babel-preset-default",
-    "@wordpress/eslint-plugin": "file:./gutenberg/packages/eslint-plugin",
+    "@wordpress/eslint-plugin": "file:gutenberg/packages/eslint-plugin",
```

Comment:
I was wondering if we could apply the same path format in the rest of packages that use `file:`.

Reply:
This was done by `npm` after I ran: `npm uninstall @wordpress/eslint-plugin && npm install ./gutenberg/packages/eslint-plugin --save-dev` to update the sub-dependencies. I did the same with other `@wordpress/` packages now and I realised maybe some of them are not necessary and uninstalled them instead 🙂